code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -106,7 +106,7 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	max_temp = 12000
 	energy_release = -350000
 	dangerous = TRUE
-	requirements = list(/datum/gas/nitrium = 500, /datum/gas/freon = 500, /datum/gas/nitrium = 800)
```

Comment:
1200 mols is quite a lot for a stupid tesla gen. Perhaps something like 500-700mols of nitrium and 100-500mols of BZ?

Reply:
changed to 500 moles nitrium, 500 moles freon, and 500 moles tritium